provider/postgis/postgis.go

```diff
@@ -814,7 +839,7 @@ func (p Provider) MVTForLayers(ctx context.Context, tile provider.Tile, layers [
 	}
 	{
 		now := time.Now()
-		err = p.pool.QueryRow(fsql).Scan(&data)
+		err = p.pool.QueryRow(context.Background(), fsql).Scan(&data)
```

Comment:
You have access to the actual context here, so we should use that instead of context.Background()